.environment/gitleaks/gitleaks-config.toml

```diff
@@ -100,6 +103,7 @@ title = "PRIME ReportStream Gitleaks Configuration"
             'FAILED Sftp upload of inputReportId xxxx to SFTPTransportType\(host=sftp', # Not a secret, part of the docs
             'gpgkey=https:\/\/packages\.microsoft\.com\/keys\/microsoft\.asc\" \| tee \/etc\/yum\.repos\.d\/azure-cli\.repo',
             'HOST=localhost',                                   # Default cred, this is allowed (too broad??)
+            'host=sftp',                                        # Default dev env sftp. not real
```

Comment:
I had to add this after I merged in master. I'm not sure how this got into master but broke my gitleaks check.